tests/saitests/sai_qos_tests.py

```diff
@@ -1645,6 +1723,7 @@ def runTest(self):
                 # as all lossy packets are now mapped to single pg 0
                 # so we remove the strict equity check, and use upper bound
                 # check instead
+                assert(1 * cell_size <= pg_shared_wm_res[pg])
```

Comment:
This check was removed as part of https://github.com/Azure/sonic-mgmt/pull/2924 because of issues seen on Mellanox.  Any reason we need this strict checking?

Reply:
No particular reason, as per my understanding the condition should hold. If it's not expected or required to be true this can be removed.

Reply:
Good to remove. we saw failures on Mellanox due to this check